Source/Core/Core/Core.cpp

```diff
@@ -422,6 +422,9 @@ static void CpuThread(Core::System& system, const std::optional<std::string>& sa
     }
   }
 
+  if (g_ActiveConfig.bDumpTextures)
```

Comment:
I wonder if this being in [VideoCommon](https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/VideoCommon/VideoBackendBase.cpp#L393) would be a better place?  Shouldn't be hard for someone to find just seems a bit odd in `Core`.  Did you consider anywhere else?

Reply:
I presume there's at least one function in VideoCommon that gets called exactly once each time emulation starts, and isn't called at any other time, but I didn't have much luck finding it.

Reply:
Yeah, it's a bit difficult to trace.  Especially with all our entry points being `Initialize`.  But yeah, there's a backend  [Initialize()](https://github.com/dolphin-emu/dolphin/blob/efc395f7f424f71f4f6ccd055a1dd5817ad8fbc8/Source/Core/Core/Core.cpp#L578) which gets called on emulation startup.  That calls `InitializeShared()` which is what I linked to originally.

Reply:
Whoops, I totally missed the link in your first post. Yeah, that's a good place for it.